dlgr/griduniverse/game_config.yml

```diff
@@ -59,92 +73,111 @@ items:
     planting_cost: 1
     portable: true
     respawn: true
+    limit_quantity: true
     sprite: "#2a9b0f,#1a6b54"
 
   # One Hour, One Life items
-  # - crossable: true
-  #   interactive: true
-  #   name: Wild Carrot Plant
-  #   item_id: 1
-  #   portable: false
-  #   spawn_rate: 0.15
-
-  # - crossable: true
-  #   interactive: true
-  #   name: Stone
-  #   item_id: 2
-  #   portable: true
-  #   spawn_rate: 0.05
-
-  # - crossable: false
-  #   interactive: true
-  #   name: Big Hard Rock
-  #   item_id: 3
-  #   portable: false
-  #   spawn_rate: 0.05
-
-  # - crossable: false
-  #   interactive: true
-  #   n_uses: 6
-  #   name: Gooseberry Bush
-  #   item_id: 4
-  #   portable: false
-  #   spawn_rate: 0.1
-
-  # - crossable: false
-  #   interactive: true
-  #   name: Sharp Stone
-  #   item_id: 5
-  #   portable: true
-  #   spawn_rate: 0.1
-
-  # - calories: 5
-  #   crossable: true
-  #   interactive: true
-  #   n_uses: 1
-  #   name: Wild Carrot
-  #   item_id: 6
-  #   portable: true
-  #   spawn_rate: 0.1
-
-  # - calories: 3
-  #   crossable: true
-  #   interactive: true
-  #   n_uses: 1
-  #   name: Gooseberry
-  #   item_id: 7
-  #   portable: true
-  #   spawn_rate: 0.1
-
-  # - crossable: false
-  #   interactive: true
-  #   n_uses: 1
-  #   name: Empty Gooseberry Bush
-  #   item_id: 8
-  #   portable: false
-  #   spawn_rate: 0.1
+  - crossable: true
+    interactive: true
+    name: Wild Carrot Plant
+    item_id: 1
+    portable: true
+    spawn_rate: 0.15
+    item_count: 2
+    sprite: "#E67E22"
+
+  - crossable: true
+    interactive: true
+    name: Stone
+    item_id: 2
+    portable: true
+    spawn_rate: 0.05
+    item_count: 5
+    sprite: "#95A5A6"
+
+  - crossable: false
+    interactive: true
+    name: Big Hard Rock
+    item_id: 3
+    portable: false
+    spawn_rate: 0
+    item_count: 2
+    sprite: "#7F8C8D"
+
+  - crossable: false
+    interactive: true
+    n_uses: 6
+    name: Gooseberry Bush
+    item_id: 4
+    portable: false
+    spawn_rate: 0.1
+    item_count: 2
+    sprite: "#8E44AD"
+
+  - crossable: false
+    interactive: true
+    name: Sharp Stone
+    item_id: 5
+    portable: true
+    spawn_rate: 0.05
+    item_count: 5
+    sprite: "#BDC3C7"
+
+  - calories: 5
+    crossable: true
+    interactive: true
+    n_uses: 1
+    name: Wild Carrot
+    item_id: 6
+    portable: true
+    spawn_rate: 0.1
+    item_count: 0
+    sprite: "#E67E22"
+
+  - calories: 3
+    crossable: true
+    interactive: true
+    n_uses: 1
+    name: Gooseberry
+    item_id: 7
+    portable: true
+    spawn_rate: 0.1
+    item_count: 0
+    sprite: "#8E44AD"
+
+  - crossable: false
+    interactive: true
+    n_uses: 1
+    name: Empty Gooseberry Bush
+    item_id: 8
+    portable: false
+    spawn_rate: 0
+    item_count: 0
+    sprite: "#9B59B6"
 
 transitions:
-  - actor_end: 5
-    actor_start: 2
+  - actor_start: 2
```

Comment:
A thought: do we need to be using integer IDs, or are we just making life hard for no reason?

Reply:
There may be some place where we're doing an `int(item_id)`, but I don't think there's anything in the code that requires integers. I'm sure it had some advantages in the original C code for OneHourOneLife though. That's just the convention Natalia used in her sample config, but we can test it out with some string ids. Using a mix will probably cause comparison errors when doing lookups, but as long as they are consistently integers or strings it should be OK.